toolkit/statemachine/src/main/kotlin/br/com/arch/toolkit/statemachine/ViewStateMachine.kt

```diff
@@ -10,9 +10,9 @@ class ViewStateMachine : StateMachine<ViewState>(::ViewState) {
 
     override fun performChangeState(state: ViewState) = with(state) {
         // Visibility
-        gones.forEach { it.visibility = GONE }
-        visibles.forEach { it.visibility = VISIBLE }
-        invisibles.forEach { it.visibility = INVISIBLE }
+        gones.forEach { it.visibility = View.GONE }
```

Comment:
Because Android studio automatically imports "import android.view.View.*" and that is a Wildcard import. And it fails the lint.

I know, awkward but I have to check it out before use such a wonderful import =P hahaha

Reply:
This is not wrong, I've just asked for curiosity actually.
To avoid the wildcard import, try to change that configuration on code style that I've sent to you.